constants/index.ts

```diff
@@ -40,7 +40,7 @@ export const previousTalks: TTalkCard[] = [
   },
   {
     id: 'augmentedreality2021',
-    description: '3D e Argumented Reality with google model-viewer',
+    description: '3D e Augmented Reality with google model-viewer',
```

Comment:
This one is a typo, not the previous version. Am I wrong?

Reply:
You are right, as in the description I fixed a typo in the previous talks' constant :)